dictionary_test.go

```diff
@@ -0,0 +1,121 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStatusLoginRequest(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	sdk := NewMockSDK(mockCtrl)
+
+	var res loginResponse
+	address := "ADDRESS"
+	password := "PASSWORD"
+	params := &statusLoginParam{
+		Address:  address,
+		Password: password,
+	}
+	sdk.EXPECT().Call(&res, "status_login", params)
+
+	response, err := statusLoginRequest(sdk, address, password)
```

Comment:
May be trivial on this case, but would be nice to test different responses, like what happens when `Call` responds with an error. Anyway as I said here is just trivial.

Reply:
I think this is the only comment I didn't address. I agree with you that it would be trivial to test an error value since sdk.Call *is* one of the return values. These tests are pretty trivial in general though, they're basically just checking that sdk.Call gets called with the right arguments. Will keep this in mind when I'm adding more complicated tests to the other files.